exercises/anagram/anagram_test.rb

```diff
@@ -3,70 +3,120 @@
 require 'minitest/autorun'
 require_relative 'anagram'
 
+# Test data version:
+# 3dfde31
 class AnagramTest < Minitest::Test
   def test_no_matches
-    detector = Anagram.new('diaper')
-    assert_equal [], detector.match(%w(hello world zombies pants))
+     # skip
+     assert_equal [], Anagram.new("diaper").match(["hello", "world", "zombies", "pants"])
   end
 
-  def test_detect_simple_anagram
-    skip
-    detector = Anagram.new('ant')
-    anagrams = detector.match(%w(tan stand at))
-    assert_equal ['tan'], anagrams
+  def test_detects_simple_anagram
+     skip
+     assert_equal ["tan"], Anagram.new("ant").match(["tan", "stand", "at"])
   end
 
-  def test_detect_multiple_anagrams
-    skip
-    detector = Anagram.new('master')
-    anagrams = detector.match(%w(stream pigeon maters))
-    assert_equal %w(maters stream), anagrams.sort
+  def test_does_not_detect_false_positives
+     skip
+     assert_equal [], Anagram.new("galea").match(["eagle"])
   end
 
-  def test_does_not_confuse_different_duplicates
-    skip
-    detector = Anagram.new('galea')
-    assert_equal [], detector.match(['eagle'])
+  def test_detects_multiple_anagrams
+     skip
+     assert_equal ["stream", "maters"], Anagram.new("master").match(["stream", "pigeon", "maters"])
   end
 
-  def test_identical_word_is_not_anagram
-    skip
-    detector = Anagram.new('corn')
-    anagrams = detector.match %w(corn dark Corn rank CORN cron park)
-    assert_equal ['cron'], anagrams
+  def test_does_not_detect_anagram_subsets
+     skip
+     assert_equal [], Anagram.new("good").match(["dog", "goody"])
   end
 
-  def test_eliminate_anagrams_with_same_checksum
-    skip
-    detector = Anagram.new('mass')
-    assert_equal [], detector.match(['last'])
+  def test_detects_anagram
+     skip
+     assert_equal ["inlets"], Anagram.new("listen").match(["enlists", "google", "inlets", "banana"])
```

Comment:
This is a personal preference of mine and non necessarily something that needs changing if you have a different opinion, but I'd rather see single quotes around strings that don't require interpolation.